common/metrics/config.go

```diff
@@ -323,6 +366,13 @@ func convertPrometheusConfigToTally(
 	}
 }
 
+func setDefaultPerUnitHistogramBoundaries(clientConfig *ClientConfig) {
+	if clientConfig.PerUnitHistogramBoundaries != nil && len(clientConfig.PerUnitHistogramBoundaries) > 0 {
```

Comment:
client config might have only overwrite one unit type

Reply:
I think we should keep it backward compatible. So if there is partial customized boundaries, it will use the customized boundaries with defined unit and others fall back to the default boundaries.

Reply:
I don't think it will break anything. It just uses different bucket boundaries which should not break any metrics. The new ones in this PR is better than the old default one. For example bytes, the old default have buckets for 0.1 byte.